Update to RiMarkov.loadFrom

CyrusSUEN opened the pull request:
check multiplier and regex, if function, then use as callback

dhowe commented:
did you test this? I don't think so...

![screen shot 2015-07-13 at 6 31 42 pm](https://cloud.githubusercontent.com/assets/737638/8647768/80d3b5ee-298d-11e5-8ee6-255f624fd841.png)

CyrusSUEN commented:
> did you test this? I don't think so...

Sorry about that. And I've just updated the commit.
I made a typo while copying the code from old commits.

But I did test this the first time I implemented this. 
https://github.com/dhowe/RiTaJS/pull/50/files (see the comment in line 1059)
